conans/model/build_info.py

```diff
@@ -132,7 +277,84 @@ def _get_cpp_info():
         return self.configs.setdefault(config, _get_cpp_info())
 
 
+class Component(object):
+
+    def __init__(self, name, root_folder):
+        self._rootpath = root_folder
+        self.name = name
+        self.deps = []
+        self._lib = None
+        self._exe = None
+        self.system_deps = []
+        self.includedirs = [DEFAULT_INCLUDE]
+        self.libdirs = [DEFAULT_LIB]
+        self.resdirs = [DEFAULT_RES]
+        self.bindirs = [DEFAULT_BIN]
+        self.builddirs = [DEFAULT_BUILD]
+        self.srcdirs = []
+        self.defines = []
+        self.cflags = []
+        self.cppflags = []
+        self.cxxflags = []
+        self.sharedlinkflags = []
+        self.exelinkflags = []
+        self._filter_empty = True
+
+    def _filter_paths(self, paths):
+        abs_paths = [os.path.join(self._rootpath, p)
+                     if not os.path.isabs(p) else p for p in paths]
```

Comment:
No need for this `if`, if the path is already absolute, the `os.path.join` will do nothing 😉

Reply:
ok. I just copied this method from the CppInfo class without caring too much. Will fix it in both places